engines/python/setup/djl_python/neuron_utils/model_loader.py

```diff
@@ -231,29 +212,31 @@ def load_hf_model(self) -> "PreTrainedModel":
 
     def load_inf2_model_from_disk(self) -> "PreTrainedModel":
         if not self.config.load_split_model:
-            logging.info(f"Saving INF2 model to {self.load_path} ...")
+            logging.info(
+                f"Saving INF2 model to {self.load_path} as split model...")
             save_pretrained_split(self.model, self.load_path)
         model_kwargs = self.get_model_specific_kwargs()
         return self._neuronx_class.from_pretrained(
             self.load_path, neuron_config=self.neuron_config, **model_kwargs)
 
     def save_split_model(self):
-        logging.info(f"Saving INF2 model to {self.split_model_path} ...")
+        logging.info(
+            f"Saving INF2 model to {self.split_model_path} as split model...")
         save_pretrained_split(self.model, self.split_model_path)
 
     def set_model_format(self):
         model_path = os.path.join(os.getcwd(), self.config.model_id_or_path)
         if os.path.isdir(model_path):
-            safetensors_index = os.path.join(model_path,
-                                             "model.safetensors.index.json")
-            self.safetensors_format = os.path.isfile(safetensors_index)
+            self.safetensors_format = any(
```

Comment:
good fix - i wonder if we need to do this anywhere else? not for this PR, but would we good to check the other backends too